app/src/main/java/at/roteskreuz/stopcorona/model/repositories/QuarantineRepository.kt

```diff
@@ -159,6 +159,11 @@ interface QuarantineRepository {
      * the red and yellow contacts.
      */
     fun observeCombinedWarningType(): Observable<CombinedWarningType>
+
+    /**
+     * Fake state where the user uploaded keys yesterday and needs to upload the missing key today
```

Comment:
This looks like a hacky way. Can you please introduce some not faking method which will do similar result in a clearer flow?

Reply:
Is it only for debug?